client/games/zombies/zombies.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python3
+
+#Font: https://www.urbanfonts.com/fonts/Arcade.htm
+
+#Built on 2018-12-15 by Richard Barnes (rbarnes.org)
+#Started:         19:53
+#Progress report: 21:30 - bullets kill zombies, zombies chase, title screen, run
+#Stopped at:      22:10 - added ammo counters, life bars, increasing hoard generation speed, losing screen
+
+import socket
+from tkinter import *
+import random
+import time
+
+WIDTH  = 45
+HEIGHT = 35
+
+title="""XXXXXXX                XX    XX
+    XX                 XX    
+   XX   XXXXXX XXX XX  XXXXX XX  XXXXX  XXXXX  
+  XX    X    X XX X XX XX  X XX XX   XX X       
+ XX     X    X XX X XX XX  X XX XXXXXXX XXXXX                 
+XX      X    X XX X XX XX  X XX XX          X
+XXXXXXX XXXXXX XX X XX XXXXX XX  XXXXX  XXXXX        
+
+                  X   X   X
+                  X    X X
+                  XXX   X
+                  X X   X
+                  XXX   X
+
+XXXXXX  XX        XX                       XX
+XX   XX           XX                       XX
+XX   XX XX  XXXXX XXXXXX  XXXXX  XX XX XXXXXX
+XXXXXX  XX XX   X XX  XX      XX XXX   XX  XX
+XX XX   XX XX     XX  XX  XXXXXX XX    XX  XX
+XX  XX  XX XX   X XX  XX XX   XX XX    XX  XX
+XX   XX XX  XXXXX XX  XX  XXXXXX XX    XXXXXX"""
+
+lose="""XX   XX    XX                            XX      
+XX   XX    XX                            XX      
+XX   XX    XX      XXXXXX XXXXX  XXXXX   XX                             
+XX   XX    XX      X    X X     XX   XX  XX                              
+XX   XX    XX      X    X XXXXX XXXXXXX                                              
+XX   XX    XX      X    X     X XX       XX                           
+ XXXXX     XXXXXXX XXXXXX XXXXX  XXXXX   XX"""
+
+def sign(x):
+  if x<0:
+    return -1
+  elif x>0:
+    return 1
+  else:
+    return 0
+
+def wrap(val,maxval):
+  if val<0:
+    return maxval-1
+  elif val==maxval:
+    return 0
+  else:
+    return val
+
+class Flaschen(object):
```

Comment:
Would it be possible to use the library directly instead of creating a second copy which would need to be maintained ?